_docs/setup/kubernetes/quick-start-gke-dm.md

```diff
@@ -0,0 +1,246 @@
+
+```
+kubectl get deployments,ing -n istio-system
+```
+```
+NAME                       DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deploy/grafana             1         1         1            1           3m
+deploy/istio-ca            1         1         1            1           3m
+deploy/istio-ingress       1         1         1            1           3m
+deploy/istio-initializer   1         1         1            1           3m
+deploy/istio-mixer         1         1         1            1           3m
+deploy/istio-pilot         1         1         1            1           3m
+deploy/prometheus          1         1         1            1           3m
+deploy/servicegraph        1         1         1            1           3m
+deploy/zipkin              1         1         1            1           3m
+```
+
+
+If you deployed BookInfo, confirm that it is also installed:
+
+
+```
+kubectl get deployments,ing
+```
+```
+NAME                    DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deploy/details-v1       1         1         1            1           3m
+deploy/productpage-v1   1         1         1            1           3m
+deploy/ratings-v1       1         1         1            1           3m
+deploy/reviews-v1       1         1         1            1           3m
+deploy/reviews-v2       1         1         1            1           3m
+deploy/reviews-v3       1         1         1            1           3m
+
+NAME          HOSTS     ADDRESS         PORTS     AGE
+ing/gateway   *         35.202.120.89   80        3m
+```
+
+Note down the IP and Port assigned to BookInfo product page. (in the example above, its ```35.202.120.89:80```.
+
+
+### Access BookInfo Application
+
+Setup environment variable for the external IP address:
+
+```
+kubectl get ingress -o wide
+```
+```
+export GATEWAY_URL=35.202.120.89
+```
+
+Verify you can access the BookInfo ```http://${GATEWAY_URL}/productpage```:
+
+![BookInfo](img/dm_bookinfo.png)
+
+Now send some traffic to it:
+```
+for i in {1..100}; do curl -o /dev/null -s -w "%{http_code}\n" http://${GATEWAY_URL}/productpage; done
+```
+
+### Verify Installed Istio plugins
+
+
+**Grafana**
+
+Setup a tunnel to Grafana
+
+```
+kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=grafana -o jsonpath='{.items[0].metadata.name}') 3000:3000 &
+```
+then 
+```
+ http://localhost:3000/dashboard/db/istio-dashboard
+```
+You should see some statistics for the requests  you just sent earlier:
+
+![Grafana](img/dm_grafana.png)
+
+For more details [About the Grafana Add-on](/docs/tasks/telemetry/using-istio-dashboard.html#about-the-grafana-add-on).
+
+
+**Prometheus**
+
+Prometheus will get installed with Grafana.  You can view the istio and appliation metrics through the console:
+
+```
+kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=prometheus -o jsonpath='{.items[0].metadata.name}') 9090:9090 &
+```
+
+View the console: at
```

Comment:
Should be:

View the console at [http://localhost:9090/graph](http://localhost:9090/graph)

Reply:
done